experiment_test.go

```diff
@@ -1,19 +1,17 @@
 package myasthurts_test
 
 import (
-	"fmt"
 	"go/parser"
 	"go/token"
 
-	"github.com/fatih/structtag"
 	myasthurts "github.com/jamillosantos/go-my-ast-hurts"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
 var _ = Describe("My AST Hurts", func() {
 
-	FContext("should parse struct", func() {
+	Context("should parse struct", func() {
```

Comment:
Use `Context` for describing the test context, not what you expect to happen (this is the `It`). For example:

```go
Describe("Categorizing book length", func() {
        Context("With more than 300 pages", func() {
            It("should be a novel", func() {
                Expect(longBook.CategoryByLength()).To(Equal("NOVEL"))
            })
        })

        Context("With fewer than 300 pages", func() {
            It("should be a short story", func() {
                Expect(shortBook.CategoryByLength()).To(Equal("SHORT STORY"))
            })
        })
    })
```

In this case, you could use the `When`:

```suggestion
	When("parsing structs", func() {
```

Reply:
Ok, i understand.